WebRoot/messages/ZaMsg_ja.properties

```
@@ -2765,3 +2765,4 @@ LBL_zimbraSieveEditHeaderEnabled = Edit Header \u30b3\u30de\u30f3\u30c9:
 LBL_zimbraSieveRejectMailEnabled = "reject" \u30a2\u30af\u30b7\u30e7\u30f3:
 NAD_AdminSieveGrouper = \u30d5\u30a3\u30eb\u30bf\u30fc\u898f\u5247
 NAD_LegalInterceptGrouper = \u30ea\u30fc\u30ac\u30eb\u30a4\u30f3\u30bf\u30fc\u30bb\u30d7\u30c8
+ERROR_FEATURE_NOT_LICENSED = \u73fe\u5728\u3001\u3053\u306e\u6a5f\u80fd\u306e\u30e9\u30a4\u30bb\u30f3\u30b9\u304c\u4ed8\u4e0e\u3055\u308c\u3066\u3044\u307e\u305b\u3093\u3002\u003c\u0062\u0072\u003e\u003c\u0062\u0072\u003e \u8a73\u7d30\u306b\u3064\u3044\u3066\u306f\u3001\u300c\u69cb\u6210 \u002d \u30b0\u30ed\u30fc\u30d0\u30eb\u8a2d\u5b9a \u002d \u30e9\u30a4\u30bb\u30f3\u30b9\u300d\u3092\u53c2\u7167\u3057\u3066\u304f\u3060\u3055\u3044\u3002
```

Please update Japanese translation
```suggestion
ERROR_FEATURE_NOT_LICENSED = \u73fe\u5728\u3001\u3053\u306e\u6a5f\u80fd\u306e\u30e9\u30a4\u30bb\u30f3\u30b9\u304c\u4ed8\u4e0e\u3055\u308c\u3066\u3044\u307e\u305b\u3093\u3002\u003c\u0062\u0072\u003e\u003c\u0062\u0072\u003e \u8a73\u7d30\u306b\u3064\u3044\u3066\u306f\u3001\u300c\u8a2d\u5b9a \u002d \u30b0\u30ed\u30fc\u30d0\u30eb\u8a2d\u5b9a \u002d \u30e9\u30a4\u30bb\u30f3\u30b9\u300d\u3092\u53c2\u7167\u3057\u3066\u304f\u3060\u3055\u3044\u3002
```

---

Thank you for updating it.